1.8.9-Forge/src/main/java/net/ccbluex/liquidbounce/features/module/modules/movement/BugUp.kt

```diff
@@ -93,6 +95,15 @@ class BugUp : Module() {
                         mc.thePlayer.fallDistance = 0F
                     }
                     "ongroundspoof" -> mc.netHandler.addToSendQueue(C03PacketPlayer(true))
+
+                    "motionteleport-flag" -> {
+                        mc.thePlayer.setPositionAndUpdate(mc.thePlayer.posX, mc.thePlayer.posY + 1f, mc.thePlayer.posZ)
+                        mc.netHandler.addToSendQueue(C03PacketPlayer.C04PacketPlayerPosition(mc.thePlayer.posX, mc.thePlayer.posY, mc.thePlayer.posZ, true))
+                        mc.thePlayer.motionY = 0.1
+
+                        MovementUtils.strafe()
+                        mc.thePlayer.fallDistance = 0f
+                    }
```

Comment:
Matrix 4.8.4

Reply:
And I tested it on Mineverse.com

Reply:
Are there other uses except for matrix? Otherwise you could call the mode `Matrix`, to make it easier for the user to choose. Anyways, we will check if it works and if we need it.